screwdriver/screwdriver_local.go

```diff
@@ -54,8 +54,8 @@ func (a localApi) JobFromID(jobID int) (job Job, err error) {
 func (a localApi) PipelineFromID(pipelineID int) (pipeline Pipeline, err error) {
 	pipeline = Pipeline{
 		ID:      0,
-		ScmRepo: ScmRepo{"screwdriver-cd/screwdriver"},
-		ScmURI:  "github.com:123456:master",
```

Comment:
Fixed to use `/sd/workspace/src/screwdriver.cd/local-build/src` as `sourceDir` in `sd-local`.